src/components/PoolInfoCard.tsx

```diff
@@ -204,6 +208,27 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
             </Box>
           ))}
         </Box>
+        <Typography mt={3}>
+          Pool address:&nbsp;
+          <Link
+            href={getMultichainScanLink(chainId, poolAddress, "address")}
```

Comment:
will all other occurrences of `getMultichainScanLink` work with this change?

Reply:
This function was used on `Toastify ` as well for showing transaction link.
If we use `https://evm.evmos.org/`, I believe, we can use `tx` for `type`